File: official/docs/java/v6/billing/fund.java
package billing;

import com.easypost.exception.EasyPostException;
import com.easypost.model.PaymentMethod;
import com.easypost.service.EasyPostClient;

public class Fund {
    public static void main(String[] args) throws EasyPostException {
        EasyPostClient client = new EasyPostClient(System.getenv("EASYPOST_API_KEY"));

        client.billing.fundWallet("2000", PaymentMethod.Priority.PRIMARY);
    }
}

File: official/docs/golang/current/billing/fund.go
package example

import (
	"github.com/EasyPost/easypost-go/v4"
)

func fund() {
	client := easypost.New("EASYPOST_API_KEY")

	_ = client.FundWallet("2000", easypost.PrimaryPaymentMethodPriority)
}
